ui/pages/settings/settings.component.js

```diff
@@ -135,11 +134,11 @@ class SettingsPage extends PureComponent {
         <div className="settings-page__header">
           <div className="settings-page__header__title-container">
             {currentPath !== SETTINGS_ROUTE && (
-              <ButtonIcon
-                ariaLabel={t('back')}
+              <Icon
+                className="settings-page__back-button"
```

Comment:
Ideally, that shouldn't happen but with ButtonIcon, I am unable to pass the color prop to the ARROW. Hence, I changed it to Icon